models/users.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection')
+
+// Unused function, but waiting for feedback as to whether this would be a better approach
+
+function checkUserExists(username) {
+  return db.query(`
+    SELECT * 
+    FROM users 
+    WHERE username = $1`, [username])
+  .then(({rows: articles})=>{
+    if(articles.length === 0) {
```

Comment:
why are these articles when we are querying the users table?

Reply:
An oversight. I had already refactored this and other models to return rows or rows[0], and then assign a more useful name in the controller.

```
  .then(({rows})=>{
    return rows
  })
```